corehq/apps/domain/forms.py

```diff
@@ -490,6 +499,13 @@ def _handle_call_limit_visibility(self):
             OperatorCallLimitSettings.CALL_LIMIT_MAXIMUM
         )
 
+    def _handle_release_mode_setting_visibility(self):
+        release_mode_setting = AppReleaseModeSetting.get_settings(domain=self.domain)
```

Comment:
We had to add a new model for this! 
Can't we store this on domain itself?

Reply:
There was a review comment in one of the previous PR to avoid adding new attributes to `Domain` class and rather create a separate model or each new functionality.

Reply:
yeah, this should be a new model (it's likely I'm the person who left the comment on the previous PR). It's not directly because of couch to sql (although I do think there's potential to split up the domain model as part of that migration), it's more just avoiding increasing the size of the domain model, which already does way too many things.